pcsx2/ShiftJisToUnicode.cpp

```diff
@@ -844,9 +844,9 @@ wchar_t ShiftJIS_ConvertChar(const u8* input, int& used)
 	}
 }
 
-wxString ShiftJIS_ConvertString( const char* src )
+std::string ShiftJIS_ConvertString( const char* src )
 {
-	wxString result;
+	std::string result;
```

Comment:
We append wchars to this, you'll need to convert to UTF-8 either on append or after
e.g.
```cpp
static void AppendUCS2(std::string& str, u16 c)
{
	if (c & 0xf800)
	{
		str.push_back(0xe0 | (c >> 12));
		str.push_back(0x80 | ((c >> 6) & 0x3f));
		str.push_back(0x80 | (c & 0x3f));
	}
	else if (c & 0xff80)
	{
		str.push_back(0xc0 | (c >> 6));
		str.push_back(0x80 | (c & 0x3f));
	}
	else
	{
		str.push_back(c);
	}
}
```